Support/Testing/Travis/after_success.sh

```diff
@@ -11,6 +11,10 @@
 
 set -eu
 
+if [[ "${TRAVIS_OS_NAME-}" = "osx" ]]; then
```

Comment:
Can you use the `TARGET` variable instead of `TRAVIS_OS_NAME`? We try to make all the scripts in `Support/Testing/Travis` usable from our actual dev hosts (although for `after_success.sh` it really doesn't matter that much).